apps/live/src/redis.ts

```diff
@@ -0,0 +1,210 @@
+import Redis from "ioredis";
```

Comment:
The ioredis docs recommend using https://github.com/redis/node-redis, do you want to switch in this PR or leave for a separate PR?